scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -108,14 +112,18 @@ def compute_config(args, cpu_flags=cpuflags(), machine=None):
     alldevices = it.chain(it.chain(string.lowercase[1:]),
                           it.product(string.lowercase, string.lowercase))
 
+    configopts = {
```

Comment:
I am a bit confused with the requirement of `nicmodel` key-value initialization since we are populating/repoulating it again anyways in the if/else block below. I would suggest not to initialize this key.